flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/typeutils/BaseRowSerializer.java

```diff
@@ -165,49 +168,33 @@ public int getArity() {
 
 	/**
 	 * Convert base row to binary row.
-	 * TODO modify it to code gen, and reuse BinaryRow&BinaryRowWriter.
+	 * TODO modify it to code gen.
```

Comment:
Remove this?

Reply:
We still need take this logical to code gen.

Reply:
I see the code gen related logic is already modified. You mean more changes are needed?

Reply:
This is a method comments, it mean move this method implement to code gen instead of using `BinaryWriter.write` and `TypeGetterSetters.get`.